pyblish/cli.py

```diff
@@ -303,4 +334,42 @@ def publish(ctx,
         click.echo(_format_time(_start, _end))
 
 
+@click.command()
+@click.argument("package", default="pyblish_qml")
+@click.pass_context
+def gui(ctx, package):
+
+    environ = os.environ.copy()
+    context = ctx.obj["context"]
+
+    with _cli_plugin(data=context.data) as plugin_path:
+        environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
+            ctx.obj["plugin_paths"] + [plugin_path]
+        )
+
+        process = subprocess.Popen(
+            [sys.executable, "-m", package + ".__main__"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=environ,
+            bufsize=1,
+            universal_newlines=True
+        )
+        while True:
```

Comment:
Also we miss out on some messages from stdout in for example pyblish-qml:
With ``for line in iter(process.stdout.readline, b""):```:
```
Using Python @ 'C:\Users\tje\Documents\conda-git-deployment\windows\miniconda\envs\temp27\python.exe'
Starting pyblish-qml
Entering state: "hidden"
Entering state: "clean"
Settings:
  HeartbeatInterval = 60
  WindowSize = [430, 600]
Entering state: "visible"
Spent 83.00 ms resetting
Spent 282.00 ms resetting
Entering state: "ready"
```
Without ``for line in iter(process.stdout.readline, b""):```:
```
Starting pyblish-qml server..
Using Python @ 'C:\Users\tje\Documents\conda-git-deployment\windows\miniconda\envs\temp27\python.exe'
Using PyQt5 @ 'None'
Starting pyblish-qml
Done, don't forget to call `show()`
Entering state: "hidden"
Entering state: "ready"
Entering state: "clean"
Entering state: "alive"
Settings:
  WindowTitle = Pyblish
  HeartbeatInterval = 60
  WindowPosition = [100, 100]
  WindowSize = [430, 600]
  ContextLabel = Context
Entering state: "visible"
Entering state: "initialising"
Spent 82.00 ms resetting
Made 0 requests during publish.
Spent 213.00 ms resetting
Made 0 requests during reset.
Entering state: "ready"
No local comment, reading from context..
```